extensions/amp-o2player/0.1/amp-o2player.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isLayoutSizeDefined} from '../../../src/layout';
+import {loadPromise} from '../../../src/event-helper';
+import {user} from '../../../src/log';
+
+class AmpO2player extends AMP.BaseElement {
+
+    /** @override */
+    preconnectCallback() {
+      this.preconnect.url(this.domain_);
+    }
+
+    /** @override */
+    isLayoutSupported(layout) {
+      return isLayoutSizeDefined(layout);
+    }
+
+    /** @override */
+    layoutCallback() {
+      const width = this.element.getAttribute('width');
+      const height = this.element.getAttribute('height');
+      const pid = this.element.getAttribute('data-pid');
+      const bcid = this.element.getAttribute('data-bcid');
+      const bid = this.element.getAttribute('data-bid');
+      const vid = this.element.getAttribute('data-vid');
+      const macros = this.element.getAttribute('data-macros');
+      const env = this.element.getAttribute('env') || 'prod';
```

Comment:
You may remove ` || 'prod'`. If it's set and equals to `stage` we use `delivery.dev.vidible.tv`, otherwise `delivery.vidible.tv`. Probably it's better to rename it to `data-env`.